src/data.seed.ts

```diff
@@ -99,13 +101,22 @@ class SeedData {
         await writeFile('./seed.data.json', JSON.stringify(this.data), 'utf8');
     }
 
-    private async createOrganization(): Promise<string> {
+    private async createOrganizationId(): Promise<string> {
```

Comment:
Delete this property, below createOrganization() is sufficient

Reply:
fixed